.github/sigstore-python-conformance

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+"""
+A wrapper to convert `sigstore-conformance` CLI protocol invocations to match `sigstore-python`.
```

Comment:
Not really sure where the right spot for this is.

It doesn't seem like it belongs in the actual package itself. But on the other hand, I want to access this wrapper from the `sigstore-conformance` tests so that when we iterate on the action, we have visibility when we're breaking basic things.

Reply:
IMO `test/integration/sigstore-python-conformance` would be a reasonable path -- that would reuse our existing test structure but shouldn't interfere with `pytest`.

If it does, we could move our current unit tests from `test/` to `test/unit.`.

Reply:
Good idea. I think that's definitely cleaner.

Reply:
[5637528](https://github.com/sigstore/sigstore-python/pull/298/commits/563752850e7d1812ca70bc8b9b0dd53bc5f31a3e)